packages/my-joy-beta/src/containers/instances/snapshots.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import ReactJson from 'react-json-view';
+import PropTypes from 'prop-types';
+import forceArray from 'force-array';
+import { compose, graphql } from 'react-apollo';
+import find from 'lodash.find';
+import get from 'lodash.get';
+
+import { ViewContainer, Title, StatusLoader, Message } from 'joyent-ui-toolkit';
+
+import SnapshotsQuery from '@graphql/snapshots.gql';
+
+const Snapshots = ({ snapshots = [], loading, error }) => {
+  const _title = <Title>Snapshots</Title>;
+  const _loading = !(loading && !forceArray(snapshots).length) ? null : (
+    <StatusLoader />
+  );
+
+  const _summary = !_loading && <ReactJson src={snapshots} />;
+
+  const _error = !(error && !_loading) ? null : (
+    <Message
+      title="Ooops!"
+      message="An error occurred while loading your instance snapshots"
```

Comment:
I think this should be a commom component because I see you are repeating this a lot , what do you think ?

Reply:
Yes, I'm aware of that. I haven't made it a reusable component because I'm sure most of those views will be very different in the near future. That means that the abstractions will need to change drastically and the reusability might not make sense anymore.

Also, I'm very aware of this now: https://www.youtube.com/watch?v=mVVNJKv9esE 😃

Reply:
Valid ! Watching that video now :)